src/salus-dbg.lds

```diff
@@ -0,0 +1,68 @@
+OUTPUT_ARCH( "riscv" )
```

Comment:
Can we reuse the existing linker scripts instead of duplicating them?

Reply:
fixed

Reply:
Not resolved?

Reply:
As noted below, doing this the cleaner way doesn't work. I'm not sure why, so I went back to a version that does.

Reply:
Some options:

- Use a genrule to make the necessary substitutions from a template linker script, and then use that linker script to link.
- Create an empty `.umode` section and copy in the contents of the U-mode binary after, e.g. with `objcopy --update-section`.
- Go back to `include_bytes`.

I really don't want to be checking in three copies of the same linker script.